src/hapi/pipelines/database/org.py

```diff
@@ -39,39 +37,77 @@ def populate(self):
             format="csv",
             file_prefix="org",
         )
+
         for row in iterator:
-            org_name = row.get("#x_pattern")
-            canonical_org_name = row.get("#org+name")
-            if not canonical_org_name:
+            canonical_name = row["#org+name"]
+            if not canonical_name:
                 continue
-            self._org_map[org_name] = row
-            self._org_map[canonical_org_name] = row
-            org_acronym = row.get("#org+acronym")
-            if org_acronym:
-                self._org_map[org_acronym] = row
+            normalised_name = normalise(canonical_name)
+            country_code = row["#country+code"]
+            acronym = row["#org+acronym"]
+            if acronym:
+                normalised_acronym = normalise(acronym)
+            else:
+                normalised_acronym = None
+            org_name = row["#x_pattern"]
+            type_code = row["#org+type+code"]
+            value = (
```

Comment:
Can you use a data class or something like that? I just find it confusing later when you are referring to like value[2] what it is exactly

Reply:
Good idea, I have used named tuples